src/pyhf/readxml.py

```diff
@@ -48,12 +48,11 @@ def import_root_histogram(rootdir, filename, path, name, filecache=None):
         h = f[name]
     except KeyError:
         try:
-            h = f[os.path.join(path, name)]
+            h = f[Path().joinpath(path, name).as_posix()]
```

Comment:
Does this not work?

```suggestion
            h = f[Path(path).joinpath(name)]
```

Otherwise, wrap it in `str`? I'm not sure about the `as_posix` because we don't test on Windows.